### DISCUSSION TITLE
UX issue with donate button on nav bar

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#112] [Feedback]
the donate button in navbar was added on my insistence for the audience segment who want to quickly come in and make a general donation.
But on further thought the other type of user who wants to donate after seeing all the projects and selecting the one in line with what they want to achieve with their donation. This type of donor will see the donate button and follow it but will only see the general donation button.

one solution is having some direct link to view campaigns/projects from the general donation section

**Comment by an-nadaa** [comment]
This is a good observation. Tbh I am not in favour of the donate button going to the general donation section. 
what i see as a better flow for both users is to redesign the [causes page](https://new.an-nadaa.com/causes?s=c) to include general donation visibly somewhere there. 
And the donate button to redirect to the causes page.
